docs/developer.md

```diff
@@ -188,6 +188,19 @@ sets up symbolic links from site-packages into the repository.
 
 First, install the dependencies by running `pip install -r build/test-requirements.txt`.
 
+Moreover, if you require visualization generated by Matplotlib during the tests,
+it is necessary to install it separately using the command:
+```
+pip install matplotlib
+```
+
+Additionally, set the LOBPCG_EMIT_DEBUG_PLOTS environment variable to 1 in
+the /tests/BUILD file or pass it via the command line. For example, for a Bazel test case:
+
+```
+bazel test //tests:cpu_tests //tests:backend_independent_tests --test_env=LOBPCG_EMIT_DEBUG_PLOTS=1
+```
+
```

Comment:
To be honest, I think adding this text is more confusing than anything: most users will not have to worry about this, and the environment variable only applies to a single file. Something like "some test targets optionally use matplotlib, so you may need to `pip install matplotlib` to run tests via `bazel`" would probably be sufficient.

Reply:
Okay, it seems reasonable that the information about the required Matplotlib library should be sufficient.